app/views/listings/_new.html.erb

```diff
@@ -0,0 +1,129 @@
+<div id="newListingModal" role="dialog" tabindex="-1" class="modal fade text-left show" style="margin-top: 20px;">
```

Comment:
New page only contains form for: home type, room type, guest count (accommodate), bedroom count, and bathroom count.
Other attributes are added during Listing management (Update)
Sample layout: 
<img width="1792" alt="Screen Shot 2021-03-22 at 6 39 06 PM" src="https://user-images.githubusercontent.com/20994464/111977737-0d29ab80-8b3e-11eb-9694-241c4755d610.png">